### PR TITLE
Export definition of Set Permission extension command

### PR DESCRIPTION
**Pull request by dontcallmedom**
Used e.g. by https://w3c.github.io/mediacapture-automation/

***
<a href="https://pr-preview.s3.amazonaws.com/dontcallmedom/permissions/pull/408.html" title="Last updated on Dec 15, 2022, 5:52 AM UTC (68b8b9d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/408/9aaca68...dontcallmedom:68b8b9d.html" title="Last updated on Dec 15, 2022, 5:52 AM UTC (68b8b9d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1217,7 +1217,7 @@ <h3 id="set-permission-command">
           </tbody>
         </table>
         <p>
-          The <dfn>Set Permission</dfn> <a>extension command</a> simulates user modification of a
+          The <dfn class="export" data-dfn-for="extension commands">Set Permission</dfn> <a>extension command</a> simulates user modification of a
```
Comment (marcoscaceres) [inline index.html:1220]: Argh, I wish WebDriver would have exported extensions command as singular.

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
@dontcallmedom, sorry, would you mind also fixing the [=Set Permission=] that broke as a result of the this change?

**Comment by dontcallmedom** [comment]
fixed

**Review by miketaylr — APPROVED** [review]
thanks!

**Pull request merged**